src/Main.java

```diff
@@ -1,5 +1,69 @@
+import java.time.LocalDate;
+
 public class Main {
+
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("задача 1");
+        int year = 2028;
+        checklLeapYear(year);
+        System.out.println("________________");
+
+        System.out.println("задача 2");
+        year = 2025;
+        int oS = 0;
+        checkAppVersion(year, oS);
+        System.out.println("________________");
+
+        System.out.println("задача 3");
+        int deliveryDistance = 95;
+        calculatingDeliveryTime(deliveryDistance);
     }
-}
\ No newline at end of file
+
+
+    //Task 1
+    public static int checklLeapYear(int checkYear) {
```

Comment:
Можно было сделать void
```suggestion
    public static void checklLeapYear(int checkYear) {
```